lib.nix

```diff
@@ -148,6 +148,7 @@ rec
                     url = val.git;
                   } // lib.optionalAttrs (val ? rev) {
                     rev = val.rev;
+                    allRefs = true;
```

Comment:
nit: The formatting looks a bit off, but I'm on the GitHub app, so this might just be me.

Reply:
This was just the github android app UI. Nevermind.